src/js/pages/More/DonateFaq.jsx

```diff
@@ -15,481 +43,428 @@ function DonateFaq () {
 
         <h2>About WeVote</h2>
 
-        <h3>What is WeVote?</h3>
-        <p>
-          WeVote is a nonpartisan, nonprofit organization committed to empowering
-          informed participation in democracy. Through innovative technology, trusted
-          data, and collaborative partnerships, we help people vote their values and
-          engage meaningfully in civic life. We Vote is where you view your ballot, see
-          endorsements from your network for all candidates and measures, and
-          collaborate with folks who share your values.
-        </p>
-        <p>
-          We Vote is a volunteer-driven movement. We rely on volunteers across the
-          country who use their engineering, design, and other skills to build WeVote.
-          We are over 100 people who have donated 12,000+ volunteer hours, including
-          90+ contributors on GitHub. We also have a small team of core staff and
-          volunteer board members. See:
-        </p>
-        <p>
-          See:
-          {' '}
-          <a
-              href="/more/faq"
-              target="_blank"
-              rel="noopener noreferrer"
-              style={{ color: DesignTokenColors.primary600 }}
-          >
-            FAQ
-          </a>
-          {' '}
-          and
-          {' '}
-          <a
-              href="/hc/en-us"
-              target="_blank"
-              rel="noopener noreferrer"
-              style={{ color: DesignTokenColors.primary600 }}
-          >
-            Help Center
-          </a>
-        </p>
-
-        <h3>Is WeVote a nonprofit organization?</h3>
-        <p>
-          Yes. WeVote is a collaboration between two nonprofits
-          {' '}
-          <a
+        <FaqItem question="What is WeVote?">
+          <p>
+            WeVote is a nonpartisan, nonprofit organization committed to empowering informed participation in democracy. Through innovative technology, trusted data, and collaborative partnerships, we help people vote their values and engage meaningfully in civic life. We Vote is where you view your ballot, see endorsements from your network for all candidates and measures, and collaborate with folks who share your values.
+          </p>
+          <p>
+            We Vote is a volunteer-driven movement. We rely on volunteers across the country who use their engineering, design, and other skills to build WeVote. We are over 100 people who have donated 12,000+ volunteer hours, including 90+ contributors on GitHub. We also have a small team of core staff and volunteer board members.
+          </p>
+          <p>
+            See:
+            {' '}
+            <StyledLink href="/more/faq">
```

Comment:
Hi @itcreativeusa unfortunately because our iOS & Android apps can't use `<a` tags to open external links, all links that open in a new tab need to use the <OpenExternalWebSite component. I'm approving this pull request, but we'll need to convert these in a future pull request.